src/scss/_base.scss

```diff
@@ -22,21 +22,24 @@ ul {
   color: #f3f3f3;
   cursor: pointer;
   padding: 0.4rem 0.8rem;
-  border-radius: 6px;
+  border-radius: 30px;
   background-color: #475ac6;
   min-width: 100px;
   font-size: 1.1rem;
+  outline: none;
   &:hover {
     color: #d6d6d6;
   }
+  margin-top: 10px;
```

Comment:
I'm not sure if I want to keep this or not. I added a `customStyles `prop to the button so that you can override anything if you want, but I thought having a defautlt margin on top and bottom would be good.